admin/index.html

```diff
@@ -379,10 +379,8 @@
             isAlive = _isAlive;
             if (isAlive || common.enabled) {
                 $('.zwave-operations').button().button('enable');
-                getComPorts(settings.usb);
-            } else {
-                $('#_usb').html('<input id="usb" class="value" value="' + settings.usb + '"/>');
-                $('#usb').change(onChange).keyup(onChange);
+                getComPorts();
+                $('#usb').val(settings.usb);
```

Comment:
That input MUST be named `usb`, because the setting in the adapter is also called usb. The default load and save routines map the settings to identically-named inputs in the admin UI.

Reply:
Not sure I understand, the input is called usb as I didn't change that. And the load and save routine works, I tested that.

Reply:
Oh wait... I misunderstood line 149. The data list has its own ID. nevermind then.